DESCRIPTION

```diff
@@ -18,6 +18,7 @@ LazyLoad: yes
 Imports:
     assertthat,
     checkmate,
+    PharmacoGx,
```

Comment:
We need to add new dependency to combo image. I've done that in https://code.roche.com/gdr/gdrshiny/-/merge_requests/3/diffs